astropy/utils/tests/test_data.py

```diff
@@ -98,19 +107,20 @@ def test_find_by_hash():
 
 
 # Package data functions
-@pytest.mark.parametrize(('filename'), ['local.dat', 'local.dat.gz', 'local.dat.bz2'])
+@pytest.mark.parametrize(('filename'), ['local.dat', 'local.dat.gz', 'local.dat.bz2', 'local.dat.xz'])
 def test_local_data_obj(filename):
```

Comment:
Just to give one more data point though, it turns out my Python (3.4) is not compiled with lzma support:

```
[GCC 4.4.7 20120313 (Red Hat 4.4.7-4)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import lzma
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/internal/1/root/usr/local/lib/python3.4/lzma.py", line 26, in <module>
    from _lzma import *
ImportError: No module named '_lzma'
>>>
```

despite this, the test is still doing *something* (I'm not sure what yet) and leaving an open file.  That might be what's happening on Travis too.